eventually-postgres/src/store.rs

```diff
@@ -188,21 +231,31 @@ impl EventStoreBuilderMigrated {
 /// [`EventStore`]: ../../eventually_core/store/trait.EventStore.html
 /// [`EventStoreBuilder`]: ../../eventually_core/store/trait.EventStoreBuilder.html
 #[derive(Debug, Clone)]
-pub struct EventStore<Id, Event> {
+pub struct EventStore<Id, Event, Tls>
+where
+    Tls: MakeTlsConnect<Socket> + Clone + Send + Sync + 'static,
+    <Tls as MakeTlsConnect<Socket>>::Stream: Send + Sync,
+    <Tls as MakeTlsConnect<Socket>>::TlsConnect: Send,
+    <<Tls as MakeTlsConnect<Socket>>::TlsConnect as TlsConnect<Socket>>::Future: Send,
```

Comment:
Do you need to specify these constraints here? Given the `impl` block already has them.